src/pages/patientView/PatientViewPage.tsx

```diff
@@ -504,7 +504,7 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                         <div className="clearfix">
                             <FeatureTitle title="Mutational Signatures" isLoading={ patientViewPageStore.clinicalDataGroupedBySample.isPending } className="pull-left" />
                             {  (patientViewPageStore.clinicalDataGroupedBySample.isComplete) && (
```

Comment:
add a check for `mutationalSignatureData` here, so `(patientViewPageStore.clinicalDataGroupedBySample.isComplete && patientViewPageStore.mutationalSignatureData.isComplete) && ...`